VMBackup/main/fsfreezer.py

```diff
@@ -128,7 +129,12 @@ def __init__(self, patching, logger, hutil):
         else:
             self.logger.log("isArm64Machine : " + str(self.isArm64Machine) + " Using x64 safefreeze binary")
             self.safeFreezeFolderPath = "safefreeze/bin/safefreeze"
-
+        
+        temp = os.path.join(os.path.dirname(os.path.realpath(__file__)), self.safeFreezeFolderPath)
```

Comment:
probably better to expose it as a helper method that can be called from other places to validate if file exists.